### PR TITLE
Remove "missing types" workaround

### PR DESCRIPTION
**Pull request by ericstj**
GenFacades has a switch that works better.

/cc @weshaggard

### CODE REVIEW

File: src/System.Runtime.InteropServices/src/System.Runtime.InteropServices.csproj [hunk 1]
```diff
@@ -6,6 +6,7 @@
     <AssemblyName>System.Runtime.InteropServices</AssemblyName>
     <OutputType>Library</OutputType>
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
+    <GenFacadesArgs Condition="'$(TargetGroup)' == 'uap101aot'">$(GenFacadesArgs) -ignoreMissingTypes</GenFacadesArgs>
```
Comment (AlexGhiondea) [inline src/System.Runtime.InteropServices/src/System.Runtime.InteropServices.csproj]: Should we turn this on for the entire repo?

Comment (ericstj) [inline src/System.Runtime.InteropServices/src/System.Runtime.InteropServices.csproj]: @weshaggard what do you think?  I'd prefer it to be intentional, myself.

**Comment by ericstj** [comment]
@dotnet-bot test Innerloop Linux ARM Emulator SoftFP Debug Cross Build

**Comment by weshaggard** [comment]
I can live with this temporarily but please file a follow-up work-item to remove this setting.

**Comment by ericstj** [comment]
@weshaggard what do you mean by "temporarily"?  All of the baselines are temporary until folks do the work to port things.  Are you saying you want something other than that here?

**Comment by weshaggard** [comment]
>@weshaggard what do you mean by "temporarily"? All of the baselines are temporary until folks do the work to port things. Are you saying you want something other than that here?

Fair. I guess I just wanted an issue that made sure we removed the "-ignoremissing" types flag.

**Comment by ericstj** [comment]
Yeah, that should be done when baselines are removed.  I'll make sure there's an issue on it and it mentions removing that flag as well as the baseline.

**Pull request merged**